gf-webapp/src/logic/rooms/roomPage/roomSlider/RoomSlider.tsx

```diff
@@ -3,83 +3,41 @@ import {Swiper, SwiperSlide} from "swiper/react";
 import styles from "./roomSlider.module.scss";
 import "swiper/css";
 import "swiper/css/navigation";
-import {Keyboard} from "swiper";
+import {Autoplay, Keyboard} from "swiper";
```

Comment:
Maybe it's not very good idea to add into  slider in the room page  autoplay parametr. This can interfere with viewing images.

Reply:
Set `disableOnInteraction` to true. Autoplay will be stopped when the user interacts with the slider.